docs/Unity-Inference-Engine.md

```diff
@@ -29,9 +29,9 @@ There are currently two supported model formats:
   [industry-standard open format](https://onnx.ai/about.html) produced by the
   [tf2onnx package](https://github.com/onnx/tensorflow-onnx).
 
-Export to ONNX is currently considered beta. To enable it, make sure
-`tf2onnx>=1.5.5` is installed in pip. tf2onnx does not currently support
-tensorflow 2.0.0 or later, or earlier than 1.12.0.
+Export to ONNX is used if using PyTorch (the default). To enable it
+while using TensorFlow, make sure `tf2onnx>=1.5.5` is installed in pip.
+tf2onnx does not currently support tensorflow 2.0.0 or later, or earlier than 1.12.0.
```

Comment:
```suggestion
while using TensorFlow, make sure `tf2onnx>=1.6.1` is installed in pip.
```
We only support tf>=1.14.0, and new tf2onnx supports tf2. My bad for not keeping this updated.